src/routing/error_cause.rs

```diff
@@ -271,30 +287,36 @@ impl ErrorClientFacing {
             Self::Other => StatusCode::INTERNAL_SERVER_ERROR,
             Self::PayloadTooLarge => StatusCode::PAYLOAD_TOO_LARGE,
             Self::RateLimited => StatusCode::TOO_MANY_REQUESTS,
-            Self::SubnetUnavailable => StatusCode::SERVICE_UNAVAILABLE,
             Self::UnknownDomain => StatusCode::BAD_REQUEST,
             Self::UpstreamError => StatusCode::SERVICE_UNAVAILABLE,
         }
     }
 
     pub fn details(&self) -> String {
         match self {
-            Self::BodyTimedOut => "Reading the request body timed out due to data arriving too slowly.".to_string(),
-            Self::CanisterError(x) => x.to_string(),
-            Self::CanisterIdIncorrect(x) => format!("The canister ID is incorrect: {x}"),
-            Self::CanisterIdNotFound => "The canister ID could not be resolved from the provided authority.".to_string(),
-            Self::Denylisted => "Access to this resource is denied due to a violation of the code of conduct.".to_string(),
-            Self::DomainCanisterMismatch => "Access to the canister is forbidden through the current gateway domain. Try accessing it through an allowed gateway domain.".to_string(),
-            Self::IncorrectPrincipal => "The principal in the request is incorrectly formatted.".to_string(),
-            Self::LoadShed => "The HTTP gateway is temporarily unable to handle the request due to high load. Please try again later.".to_string(),
-            Self::MalformedRequest(x) => x.to_string(),
-            Self::NoAuthority => "The request is missing the required authority information (e.g., Host header).".to_string(),
-            Self::Other => "Internal Server Error".to_string(),
-            Self::PayloadTooLarge => "The payload is too large.".to_string(),
-            Self::RateLimited => "Rate limit exceeded. Please slow down requests and try again later.".to_string(),
-            Self::SubnetUnavailable => "The subnet is temporarily unavailable due to maintenance or an ongoing upgrade. Please try again later.".to_string(),
-            Self::UnknownDomain => "The requested domain is not served by this HTTP gateway.".to_string(),
-            Self::UpstreamError => "The HTTP gateway is temporarily unable to process the request. Please try again later.".to_string(),
+            Self::BodyTimedOut => "Reading the request body timed out due to data arriving too slowly.".into(),
+            Self::CanisterNotFound => "The requested canister does not exist.".into(),
```

Comment:
to me this message sounds the same as `CanisterRouteNotFound` is it intentional?

Reply:
Yeah, but idea behind `CanisterRouteNotFound` that we can't find a canister range. I'll try to rephrase it.

Reply:
I've just removed `CanisterRouteNotFound` since for user it doesn't matter much